tests/static/notifications.spec.ts

```diff
@@ -296,20 +287,14 @@ test.describe("Extension triggers a notification when a page form is submitted w
             ),
           });
 
-          const notificationLocator = await testPage
+          const notificationLocator = testPage
```

Comment:
I believe we still need this `await`

```suggestion
          const notificationLocator = await testPage
```

Reply:
we can try it; most of the awaits were added out of caution anyway, but your reasoning makes sense.